### PR TITLE
Add Supported Platforms topic

### PR DESCRIPTION
**Pull request by willihay**
Signed-off-by: William Hayward <wilhayw@amazon.com>

## Change summary

Adds a new topic on Supported Platforms in O3DE, including information on host platforms and target platforms.

https://deploy-preview-1282--o3deorg.netlify.app/docs/welcome-guide/supported-platforms/

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/welcome-guide/supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+### Target platforms
+
+| Target Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Creating a release layout for Windows](/docs/user-guide/packaging/windows-release-builds/)</li></ul> |
+| Linux | <ul><li>[Platform development support for Linux](/docs/user-guide/platforms/linux/)</li></ul> |
+| Android | <ul><li>[Platform development support for Android](/docs/user-guide/platforms/android/)</li></ul> |
+
+{{< note >}}
+Documentation for **macOS** and **iOS** as target platforms is expected at a later date.
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:35]:
```suggestion
Documentation for **macOS** and **iOS** as target platforms will be added at a later date.
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Support for developing on **macOS** is in an experimental stage. Documentation for developing on a Mac will be added at a later date.
+{{< /note >}}
+
+### Target platforms
+
+| Target Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Creating a release layout for Windows](/docs/user-guide/packaging/windows-release-builds/)</li></ul> |
+| Linux | <ul><li>[Platform development support for Linux](/docs/user-guide/platforms/linux/)</li></ul> |
+| Android | <ul><li>[Platform development support for Android](/docs/user-guide/platforms/android/)</li></ul> |
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:32]:
```suggestion
| Android | <ul><li>[Platform development support for Android](/docs/user-guide/platforms/android/)</li></ul> |
| iOS | |
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+**Open 3D Engine (O3DE)** supports several _host platforms_ for creating and developing games and simulations, and _target platforms_ on which these projects can run.
+
+### Host platforms
+
+| Host Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Windows prerequisites and configuration](requirements/#microsoft-windows)</li><li>[Installing O3DE for Windows](setup/installing-windows)</li><li>[Creating projects from the CLI on Windows](create/creating-projects-using-cli/creating-windows/)<sup>1</sup></li><li>[Building O3DE from source on Windows](setup/setup-from-github/building-windows/)</li></ul> |
+| Linux | <ul><li>[Linux prerequisites and configuration](requirements/#linux)</li><li>[Installing O3DE for Linux](setup/installing-linux)</li><li>[Creating projects from the CLI on Linux](create/creating-projects-using-cli/creating-linux/)<sup>1</sup></li><li>[Building O3DE from source on Linux](setup/setup-from-github/building-linux/)</li></ul> |
+
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:17]:
```suggestion
| macOS | |

```

File: content/docs/welcome-guide/supported-platforms.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+**Open 3D Engine (O3DE)** supports several _host platforms_ for creating and developing games and simulations, and _target platforms_ on which these projects can run.
+
+### Host platforms
+
+| Host Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Windows prerequisites and configuration](requirements/#microsoft-windows)</li><li>[Installing O3DE for Windows](setup/installing-windows)</li><li>[Creating projects from the CLI on Windows](create/creating-projects-using-cli/creating-windows/)<sup>1</sup></li><li>[Building O3DE from source on Windows](setup/setup-from-github/building-windows/)</li></ul> |
+| Linux | <ul><li>[Linux prerequisites and configuration](requirements/#linux)</li><li>[Installing O3DE for Linux](setup/installing-linux)</li><li>[Creating projects from the CLI on Linux](create/creating-projects-using-cli/creating-linux/)<sup>1</sup></li><li>[Building O3DE from source on Linux](setup/setup-from-github/building-linux/)</li></ul> |
+| macOS | |
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:17]:
```suggestion
| macOS | Support for developing on **macOS** is in an experimental stage. Documentation coming in 2022. |
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+<sup>1</sup> Platform-independent documentation on creating projects using **Project Manager** can be found [here](create/creating-projects-using-project-manager).
+
+For minimum hardware requirements for development on host platforms, refer to [System requirements](requirements).
+
+{{< note >}}
+Support for developing on **macOS** is in an experimental stage. Documentation for developing on a Mac will be added at a later date.
+{{< /note >}}
+
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:26]:
```suggestion
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+### Target platforms
+
+| Target Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Creating a release layout for Windows](/docs/user-guide/packaging/windows-release-builds/)</li></ul> |
+| Linux | <ul><li>[Platform development support for Linux](/docs/user-guide/platforms/linux/)</li></ul> |
+| Android | <ul><li>[Platform development support for Android](/docs/user-guide/platforms/android/)</li></ul> |
+| iOS | |
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:34]:
```suggestion
| Android | <ul><li>[Platform development support for Android](/docs/user-guide/platforms/android/)</li></ul> |
| macOS | Support for macOS as a target platform is in an experimental stage. Documentation coming in 2022. |
| iOS | Support for iOS as a target platform is in an experimental stage. Documentation coming in 2022. |
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+{{< note >}}
+Documentation for **macOS** and **iOS** as target platforms will be added at a later date.
+{{< /note >}}
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:38]:
```suggestion
```

File: content/docs/welcome-guide/supported-platforms.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+| --- | --- |
+| Microsoft Windows | <ul><li>[Windows prerequisites and configuration](requirements/#microsoft-windows)</li><li>[Installing O3DE for Windows](setup/installing-windows)</li><li>[Creating projects from the CLI on Windows](create/creating-projects-using-cli/creating-windows/)<sup>1</sup></li><li>[Building O3DE from source on Windows](setup/setup-from-github/building-windows/)</li></ul> |
+| Linux | <ul><li>[Linux prerequisites and configuration](requirements/#linux)</li><li>[Installing O3DE for Linux](setup/installing-linux)</li><li>[Creating projects from the CLI on Linux](create/creating-projects-using-cli/creating-linux/)<sup>1</sup></li><li>[Building O3DE from source on Linux](setup/setup-from-github/building-linux/)</li></ul> |
+| macOS | Support for developing on **macOS** is in an experimental stage. Documentation coming in 2022. |
+
+<sup>1</sup> Platform-independent documentation on creating projects using **Project Manager** can be found [here](create/creating-projects-using-project-manager).
+
+For minimum hardware requirements for development on host platforms, refer to [System requirements](requirements).
```
Comment (chanmosq) [inline content/docs/welcome-guide/supported-platforms.md:21]:
```suggestion
For minimum hardware requirements for development on host platforms, refer to [System Requirements](requirements).
```

**Review by chanmosq — APPROVED** [review]
Helpful navigation! 

Minor suggestion to capitalize the title of a page in a reference.

File: content/docs/welcome-guide/supported-platforms.md (hunk 8 above)
Comment (willihay) [reply content/docs/welcome-guide/supported-platforms.md:21]: Funny you mention that, I kind of prefer this capitalization on links too, even though I've seen editors change this to use sentence casing.

File: content/docs/welcome-guide/supported-platforms.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+**Open 3D Engine (O3DE)** supports several _host platforms_ for creating and developing games and simulations, and _target platforms_ on which these projects can run.
+
+### Host platforms
+
+| Host Platform | Platform-specific Documentation |
+| --- | --- |
+| Microsoft Windows | <ul><li>[Windows prerequisites and configuration](requirements/#microsoft-windows)</li><li>[Installing O3DE for Windows](setup/installing-windows)</li><li>[Creating projects from the CLI on Windows](create/creating-projects-using-cli/creating-windows/)<sup>1</sup></li><li>[Building O3DE from source on Windows](setup/setup-from-github/building-windows/)</li></ul> |
+| Linux | <ul><li>[Linux prerequisites and configuration](requirements/#linux)</li><li>[Installing O3DE for Linux](setup/installing-linux)</li><li>[Creating projects from the CLI on Linux](create/creating-projects-using-cli/creating-linux/)<sup>1</sup></li><li>[Building O3DE from source on Linux](setup/setup-from-github/building-linux/)</li></ul> |
+| macOS | Support for developing on **macOS** is in an experimental stage. Documentation coming in 2022. |
```
Comment (willihay) [inline content/docs/welcome-guide/supported-platforms.md:17]:
```suggestion
| macOS | Support for developing on macOS is in an experimental stage. Documentation coming in 2022. |
```

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**